### DISCUSSION TITLE
LinkFree Logo

### DISCUSSION THREAD

**Discussion by rohan-kulkarni-25** [EddieHubCommunity/LinkFree#206] [Ideas]
- We will be needing a logo for our linkfree project 
- You can attach logo in form of  png in comments. 
- Try to make the logo unique, attractive and relatable to project. 
- You can use canva to make logo ( Please don't copy from somewhere )

**Comment by ShlokMohanty** [comment]
is it a graphic designing part? @rohan-kulkarni-25

**Comment by Akshu-on-github** [comment]
<img src="https://user-images.githubusercontent.com/61582763/133870264-f0c9c9e5-9982-407b-8384-7ea762ca3bd9.png" align="right">

Here are a few minimalist icons to draw inspiration from, though the first two may directly be used as the project trademark. Anti-clockwise, from the bottom:

- [**css.gg**](https://github.com/astrit/css.gg)
  - Author: Astrit
  - MIT Licence (can use as Trademark)

- [**Clarity**](https://github.com/vmware/clarity)
  - Author: VMware
  - License: MIT (can use as Trademark)

- [**IconPark**](https://github.com/bytedance/IconPark)
  - Author: ByteDance
  - License: Apache 2.0 (can't use as Trademark)

**Reply by ShlokMohanty** [reply]
@Akshu-on-github this a great idea we can think of more icons but yeah these ones are not bad

**Reply by NS007-dev** [reply]
As perhaps an idea, shouldn't the icon follow the EddieHub color theme if the logo is in color, if it is in black and white, then it doesn't matter?

Here is the theme:
![image](https://user-images.githubusercontent.com/70807500/133881132-1d02efdf-5227-458a-a7b6-ba679ba348b3.png)

Thanks.

**Reply by AbhijithGanesh** [reply]
Points noted, closing the other Issue.

**Reply by SaraJaoude** [reply]
@CodingSpecies I think having the logo following the EddieHub colour theme is a great idea! Good to keep everything consistent

**Comment by eddiejaoude** [comment]
Great ideas and discussions 💥

**Comment by veerbal1** [comment]
Any UI designer is here, who can contribute to create a logo, so that we can start implementing it..

**Reply by Akshu-on-github** [reply]
Maybe we should share this discussion under #help and/or #ui on discord?

**Reply by atharvashirdhankar** [reply]
Can I try some design for Logo

**Reply by Akshu-on-github** [reply]
Of course @StarTrooper08!

**Comment by atharvashirdhankar** [comment]
Hello everyone I'll collaborating with @sakshipawar001 for designing Logo for LinkFree.

**Reply by sakshipawar001** [reply]
I'm glad to collaborate with you. Creating a logo by combining coding and designing will be a great fun experience !

**Comment by atharvashirdhankar** [comment]
@sakshipawar001 is designing Digital Art and UI/UX from about 2-3 years .

Few of the designs by @sakshipawar001
https://www.behance.net/sakshipawar3

**Reply by sakshipawar001** [reply]
Hello everyone. Thanks @StarTrooper08 for addressing my work ! Though I don't have much experience of UI UX designing but I have been creating digital artworks since a long time. I hope I can contribute something to the open source community for creating this logo.

**Reply by atharvashirdhankar** [reply]
Welcome. 
If you have any idea regarding making logo for LinkFree . Go ahead and share it here in discussion.

**Comment by suitupstan** [comment]
how about reaching out to the opensourcedesign community?

**Reply by Akshu-on-github** [reply]
Ooh, that sounds interesting 👀

**Comment by Yash621** [comment]
I have found this logo for linkfree ,can anyone review it ?
![favicon](https://user-images.githubusercontent.com/58719389/135716454-35abe8ce-036f-4e1a-a8f3-d82d2dc259d3.png)

**Reply by NS007-dev** [reply]
Is this ok for us to use, like the license and copyright? @Yash621

**Reply by Yash621** [reply]
yups

**Reply by Akshu-on-github** [reply]
Could you share the link to the source? We'll need to review the licence + give proper attribution

**Reply by Akshu-on-github** [reply]
But it looks pretty cool!

**Reply by NS007-dev** [reply]
It does seem to link with the EddieHub Colors and resembles the meaning of the project! Looks nice.

**Reply by eddiejaoude** [reply]
It is probably best we design our own to avoid an copyright issues or conflicts with someone else using it too

**Reply by GoodnessEzeh** [reply]
That's a good idea. Making a customized logo for the app speak volumes of how original the app is. In addition, I suggest the logo is designed such that it conveys meaning and the ideology of the app. We shouldn't just get a logo just for the sake of it.

**Comment by Yash621** [comment]
@CodingSpecies  https://chipapk.com/app/6975620/ here is the source,it's something like linkfree only ,but I don't see any licences ,so I think we can use it .

**Reply by NS007-dev** [reply]
Is this logo not already being used on the app seen on the source? @Yash621

**Reply by eddiejaoude** [reply]
if no license we cannot use it

**Comment by Yash621** [comment]
actually it was there previous logo ,but now they have changed the color combination slightly and also I can't see any licences also , so what I was thinking is if we remove the small orange dot on the logo,we should not face any problem then.
this is their current logo :https://apps.apple.com/us/app/instabio-link-in-bio/id1455604586

**Reply by Akshu-on-github** [reply]
I see... I understand that it's no longer in use. However, it _was_ once a part of the branding for another product so I believe that we might require permission

@eddiejaoude, may I ask your thoughts on this?

**Reply by Yash621** [reply]
@CodingSpecies any thoughts about this ?

**Reply by NS007-dev** [reply]
@Yash621 It is a nice logo, what we need to see:

- [ ] If we have permission.
- [ ] Also I think it has a white background to it, as it's not transparent. I think that might not look right.

**Reply by eddiejaoude** [reply]
it is best to create our own custom logo

@Yash621 thank you for the suggestion, please could you try to keep the same topic in a similar thread so that it is easier to follow rather than starting new threads each time

**Comment by TakshakRamteke** [comment]
Hey all :wave:, I just made a few rough sketches for the logo designs, Also as please consider that these are just sketches so the illustrations will look considerably different from what you see here 

![drawings](https://user-images.githubusercontent.com/44837950/136157882-053e3c78-ad88-4f47-95a5-78e73b535ddd.jpg)

**Reply by TakshakRamteke** [reply]
If you would Like to see the illustrations of any of these please feel free to comment the number here and i would love to make the illustration for that one also let me know if i should make some more design sketches

i personally like no. 5, 9 & 10

**Reply by Akshu-on-github** [reply]
I prefer 1 actually - took me a second to recognise the L and F in the 1st sketch, but once I did, I thought it was pretty clever!

I think the other sketches are cool too, but keep in mind that we'll be looking into a favicon too - they'd lose a lot of details when brought down to ~20px

**Reply by NS007-dev** [reply]
I think 1 is good too as I like the way you played around with the letters. @TakshakRamteke

**Reply by TakshakRamteke** [reply]
Sure, I made an illustration for 1 [Here's](https://drive.google.com/drive/folders/1N335I1AlVwVFIFdamU4b-uNVavjFthG0?usp=sharing) the link to the full size logo and the favicon as well please let me know how it is

**Reply by NS007-dev** [reply]
@TakshakRamteke I cant seem to access the link, please can you send a screenshot of it perhaps instead, I am excited to see it.

**Reply by TakshakRamteke** [reply]
![logo1](https://user-images.githubusercontent.com/44837950/136414840-e3dd707e-bfe5-40a3-a933-d1d97ea81adf.png)

@CodingSpecies Sorry about that 😅

**Reply by NS007-dev** [reply]
> 
> 
> ![logo1](https://user-images.githubusercontent.com/44837950/136414840-e3dd707e-bfe5-40a3-a933-d1d97ea81adf.png)
> 
> @CodingSpecies Sorry about that 😅

@sakshipawar001 this looks really cool, the color scheme matches the EddieHub theme, and I clearly see the LF, just see what others think! I like it a lot.

Feedback:
- Perhaps make it transparent, so there is no grey background.
- Simplify it down a little bit, so you can clearly see the fact it is meant to resemple some sort of linking squares in the image. 

Over all, I think this is really nice!

**Reply by TakshakRamteke** [reply]
i think we should keep the icon in a little circle/square because if someone is using light mode (white theme) the white part of logo might not come through, what do you think ? thanks for the feedback tho will try to simplifying it :slightly_smiling_face:

**Reply by SaraJaoude** [reply]
@TakshakRamteke I really like how you shared the journey from the sketching to the proposed logo idea

**Reply by eddiejaoude** [reply]
wow the journey is super interesting 💥

**Reply by EmmaDawsonDev** [reply]
It’s nice and bold, however for a logo it’s got too much detail at the moment. It’s given me an idea though, will be back when I’ve done a sketch…

**Reply by NS007-dev** [reply]
@emmalearnscode looking forward to see what you have got for this project too!

**Comment by sakshipawar001** [comment]
Hello. I took reference from the different logos on the internet and tried to create a unique logo design out of it. Here's the design made by me using Inkscape. Please do review it and give your suggestions. If you all like it, I'll move on to the color part. @rohan-kulkarni-25

![LinkFree3](https://user-images.githubusercontent.com/44046769/136161260-adb0995c-7ac7-4b47-a889-77555898aff4.png)

**Reply by NS007-dev** [reply]
> 
> 
> Hello. I took reference from the different logos on the internet and tried to create a unique logo design out of it. Here's the design made by me using Inkscape. Please do review it and give your suggestions. If you all like it, I'll move on to the color part. @rohan-kulkarni-25
> 
> ![LinkFree3](https://user-images.githubusercontent.com/44046769/136161260-adb0995c-7ac7-4b47-a889-77555898aff4.png)

@sakshipawar001 this looks cool!

**Reply by atharvashirdhankar** [reply]
I m excited to see this logo after adding colours to it👀✨

**Reply by NS007-dev** [reply]
Yes, this might be a very good logo.

**Reply by suitupstan** [reply]
Yeah, but if we were named "LinkTree" since it resembles more of a tree. imho We should have a logo that could symbolise links and/or the polywork features

**Reply by atharvashirdhankar** [reply]
![IMG-20211005-WA0017.jpg](https://user-images.githubusercontent.com/72031540/136414675-d6742b23-a439-4db6-841b-6bec1a8a3b42.jpg)

Designed by : {@sakshipawar001}

@CodingSpecies please review.

**Reply by NS007-dev** [reply]
This logo here made by  @sakshipawar001 looks awesome too. I like the linking of the hands, and the rectangles, and the color schemes fit!

**Comment by Yash621** [comment]
@CodingSpecies  I have made the background transparent and have also changed it's color ,now I don't think there will be any permissions issues . could you please review it ?
![download](https://user-images.githubusercontent.com/58719389/136216599-144be5b7-f080-479c-b5e1-e7738f28884f.png)

**Reply by suitupstan** [reply]
Could it be a higher res? I can see already see the pixels on the corners

**Reply by NS007-dev** [reply]
Thank you for making it transparent like @krishguptadev said just needs to be higher resolution @Yash621

**Reply by TakshakRamteke** [reply]
Sorry but i Just wanted to point out that... Actually in some cases we are not allowed to manipulate the logo, e.g. : in case of ieee student branch the student branch can only use the ieee logo in a predefined guidelines and anything else is considered the wrong use/ violation of the guidelines 

Also the app ( instabio ) that uses this logo has a full blown pricing model so i guess there must be some catch here

**Reply by Yash621** [reply]
I don't think so ,I have have gone through all their guidelines and I did n't find anything there like this

**Reply by suitupstan** [reply]
If its a logo of another app, we can't use it

**Reply by TakshakRamteke** [reply]
[Heres](https://instabio.cc) a link to the instabio website you can see it for yourself, iam just a little concerned for using this logo because the app has a pricing model associated to it (and as all the rights are reserved that includes the logo as well 😬)

**Reply by NS007-dev** [reply]
Yes, @TakshakRamteke this favicon is already being used :
![image](https://user-images.githubusercontent.com/70807500/136234682-2d644d74-abe6-4f62-8e37-c8cc8397df65.png)

**Reply by Yash621** [reply]
then are we not going with it ? @CodingSpecies

**Reply by suitupstan** [reply]
Yeah, we are not gonna go with some other company/app's logo

**Reply by NS007-dev** [reply]
> 
> 
> then are we not going with it ? @CodingSpecies

@Yash621 we can't go with this as it is being used, unfortunately. I like the research you did to find it, and maybe we can use some parts to implement a new logo? Thank you.

**Reply by Yash621** [reply]
@CodingSpecies sure we can do this :)

**Comment by RiddhiDamani** [comment]
Hey all, here's a logo suggestion for LinkFree. Abiding by the color scheme of EddieHub Color Scheme. Feel free to share your reviews on this. 

![LinkFree (2)](https://user-images.githubusercontent.com/19336011/136988679-2be696d4-6276-4e1c-b2b8-cbfebfc3b0c1.png)

**Reply by TakshakRamteke** [reply]
Hey Riddhi, Just a little suggestion, maybe try using a lighter color for the black arc just so the transition between the two colors is a little bit smooth know what i mean ?

**Reply by EmmaDawsonDev** [reply]
Nice design. What’s the reasoning behind the serif on the black link? Can you explain a little of your thought process of how you got to this design?

**Comment by NS007-dev** [comment]
### **So far we have these ideas. There is a few, so I thought we could gather them up and see the ones we like the most, just react by the corresponding emojis for each logo:** 
### ɪꜰ ʏᴏᴜ ᴡᴏᴜʟᴅ ʟɪᴋᴇ ʏᴏᴜʀ ʟᴏɢᴏ ᴀᴅᴅɪɴɢ, ᴘʟᴇᴀꜱᴇ ᴀᴅᴅ ᴛʜᴇ ɪᴍᴀɢᴇ ᴛᴏ ᴛʜɪꜱ ᴄʜᴀᴛ ᴛʜʀᴇᴀᴅ. ᴛʜᴀɴᴋ ʏᴏᴜ.

| Emoji | Author | Image |
| ----- | ------- | ------- |
|  <h1>👍</h1> | @TakshakRamteke | <img src="https://user-images.githubusercontent.com/44837950/136414840-e3dd707e-bfe5-40a3-a933-d1d97ea81adf.png" height=70 /> |
| <h1>😕</h1> | @TakshakRamteke, please specify which one in the thread below |  <img src="https://user-images.githubusercontent.com/44837950/137635655-b84c5b27-6a8d-482f-9ed2-645db7fee1bb.png" height=70 /><img src="https://user-images.githubusercontent.com/44837950/139289342-0ac02e41-e788-4e4f-abc2-731e833f4cf0.png" height=70 /> <img src="https://user-images.githubusercontent.com/44837950/137642332-8d961f38-4121-435a-813b-79f4c4cf3893.png" /> <img src="https://user-images.githubusercontent.com/44837950/139065368-db04de1e-74b6-4c82-8f9b-a8321c6db197.png" height=70 /> |
| <h1>🎉</h1> | Originally by @TakshakRamteke , simplified by @emmalearnscode | <img src="https://user-images.githubusercontent.com/57045550/137634982-4b17bac3-7221-4932-a506-1ccbcd96b3c5.jpeg" height=70 /> <img src="https://user-images.githubusercontent.com/57045550/137634979-94047b25-57b3-4f66-8cb6-7cabc7f9ea52.jpeg" height=70 /> |
| <h1>😄</h1> | @sakshipawar001 | <img src="https://user-images.githubusercontent.com/72031540/137471016-2ed07543-1c9f-4843-8198-1cbe84d33918.png" height=70 /> |
|  <h1>❤️ </h1> | @RiddhiDamani | <img src="https://user-images.githubusercontent.com/19336011/136988679-2be696d4-6276-4e1c-b2b8-cbfebfc3b0c1.png" height=70 /> |
|  <h1>🚀</h1> | @TakshakRamteke, please specify which design | <img src="https://user-images.githubusercontent.com/44837950/136157882-053e3c78-ad88-4f47-95a5-78e73b535ddd.jpg" height = 170 /> |
| <h1> 👎 </h1> | @krishguptadev, please specify which one |  ![image](https://user-images.githubusercontent.com/70807500/141693840-e429e3e4-6dfb-4967-84f0-4bd5ab6dd6ce.png)   ![image](https://user-images.githubusercontent.com/70807500/141693895-9271efd1-5595-4dd8-8c78-6c6bc1381dc6.png) |

**Reply by sh-himanshu** [reply]
[🚀 (9) ] seems interesting, can he make it more minimalistic?

**Reply by NS007-dev** [reply]
Yes @sh-himanshu that is an interesting one. I agree maybe we could make it a bit minimalistic @TakshakRamteke

**Reply by TakshakRamteke** [reply]
I might be able to make a bit minimal, but instead of using it for a logo how about we use it as a banner for the README or maybe even better we can make it into a badge for all the eddiehubers something that we can show on on our socials maybe like "Hey iam using LinkFree " kind of thing or maybe "Hey i Contributed to LinkFree" Something like that. What do you people think ?

**Reply by NS007-dev** [reply]
Actually when I think about it, it seems like a really
Good sort of socials badge or sticker @TakshakRamteke

**Reply by NS007-dev** [reply]
That’s where 5 and 10 could also be helpful. For social media and badges

**Reply by TakshakRamteke** [reply]
Let's see what the others have to say about this, and in the mean time i will try making them ( with some cool mock-ups maybe ) sounds good ?

**Reply by NS007-dev** [reply]
Yes, thank you @TakshakRamteke

**Reply by Akshu-on-github** [reply]
I think we should create a new poll

**Reply by NS007-dev** [reply]
@Akshu-on-github is the table design better? 
I don't think we should make a new poll, as we have already had votes.

**Reply by Akshu-on-github** [reply]
@CodingSpecies - much better!

**Reply by suitupstan** [reply]
I am waiting for others to say about this...

**Reply by NS007-dev** [reply]
@eddieHubcommunity/linkfree is there any logos you like? I think this discussion has being going on for a while, and I think we may need some answers.
- Once some designs are narrowed down, we can solely work on them, which would progress smoothly to having a final logo, which we can use for this project.
- At the top of this thread there is a poll, and the most popular logos are: 
<img src="https://user-images.githubusercontent.com/70807500/143611322-0c094377-16d4-44b2-8544-aa86b44aefde.png" height=200 />
by @TakshakRamteke . Also:<img src="https://user-images.githubusercontent.com/44837950/136414840-e3dd707e-bfe5-40a3-a933-d1d97ea81adf.png" height=70 /> also by @TakshakRamteke .

Thank you.

**Comment by atharvashirdhankar** [comment]
![LinkFreeLogo_Final.png](https://user-images.githubusercontent.com/72031540/137471016-2ed07543-1c9f-4843-8198-1cbe84d33918.png)

More clear one

Design Credit : { @sakshipawar001 }

**Reply by atharvashirdhankar** [reply]
Any suggestion ?

**Reply by suitupstan** [reply]
What does that handshake represent?

**Reply by atharvashirdhankar** [reply]
Professional network 
That's what I called it😅

**Reply by suitupstan** [reply]
Seems good but might choose some better colours

**Reply by atharvashirdhankar** [reply]
Yes we are working on that part and adding the background to make it more like Logo.

**Reply by atharvashirdhankar** [reply]
@eddiejaoude I would like to know how this logo is looking, it's still not in complete state but yes @sakshipawar001 working on it to make it better.

**Reply by eddiejaoude** [reply]
There are so many good logos 🎉 

It looks good, I think the hands looks a bit too formal, I wonder if something else could be used instead?

**Reply by EmmaDawsonDev** [reply]
Also, think about how this would look as a small icon, imaging Twitter icon, will it work that small? Is there some detail that could be removed to make it work better?

**Reply by sakshipawar001** [reply]
@emmalearnscode I agree with you emma. While using it as small icon, the details won't be visible. Removing it will again make a common link logo design. So I thought of a new design inspired from you and @TakshakRamteke . Please do review it and give your suggestions.

**Comment by EmmaDawsonDev** [comment]
I was inspired by @TakshakRamteke ‘s design but wanted to make it a bit more minimal. These are just a couple of quick sketches and the colours aren’t quite right (didn’t have the orange code when I was sketching) but maybe something to develop further.
![124824EC-C480-479F-879B-9C2274BA7B22](https://user-images.githubusercontent.com/57045550/137634979-94047b25-57b3-4f66-8cb6-7cabc7f9ea52.jpeg)
![B6C81E7D-D068-4F95-8074-2C987D461049](https://user-images.githubusercontent.com/57045550/137634982-4b17bac3-7221-4932-a506-1ccbcd96b3c5.jpeg)

**Reply by NS007-dev** [reply]
@emmalearnscode these are great designs evolving from @TakshakRamteke idea! 
I prefer the white one, in my opinion! 
I will add these now to the list of logos on the vote!
Thank you so much!

**Reply by NS007-dev** [reply]
@emmalearnscode the concept with @TakshakRamteke design is how they look as the the letters are in a link, like the name `LinkFree`. Maybe if you could incorporate that, but keep it minimal?
Overall, I like how you took it and simplified it!
Thanks for your collaboration! Let's see what others say!

**Reply by TakshakRamteke** [reply]
Great work @emmalearnscode, These look really great :sparkles: 
BTW which tool did you use (just curious to know)

**Reply by EmmaDawsonDev** [reply]
@TakshakRamteke I sketched it in procreate, it needs a bit of cleanup but I like procreate for quickly getting ideas sketched out.

**Reply by TakshakRamteke** [reply]
procreate on ipad right :drooling_face: ?, Now I'm really looking forward to your designs

**Comment by TakshakRamteke** [comment]
Hey Everyone :wave: , How's This one ?

![Linkfree-trial](https://user-images.githubusercontent.com/44837950/137635655-b84c5b27-6a8d-482f-9ed2-645db7fee1bb.png)

P.S : Someone needs to help me with coloring these logos 🥲 ig i'm really bad at it

**Reply by EmmaDawsonDev** [reply]
I really like this one a lot @TakshakRamteke. My only suggestion is to remove the coloured borders and try it all black or with the outside black and the bar across the center in orange

**Reply by NS007-dev** [reply]
@TakshakRamteke I like this logo! Though it took me a while to realise that this is an L and an F: 
![image](https://user-images.githubusercontent.com/70807500/137636374-42e7a48d-399f-483a-a051-60d310614847.png)

I would say that each letter should be a different color, so it is easier to see the 2 letters: The L should be how it is now, and the F should be a lighter tone. I really like this! Thank you

**Reply by TakshakRamteke** [reply]
@emmalearnscode and @CodingSpecies thanks for the suggestions, although I think we should really keep this one mono-chrome as it is a Typeface logo e.g : 
![trialx](https://user-images.githubusercontent.com/44837950/137642332-8d961f38-4121-435a-813b-79f4c4cf3893.png)

P.S : The colors used are the colors from mascot except for the white color

**Reply by NS007-dev** [reply]
Yes, @TakshakRamteke I think number 4 is what I like the most!

**Reply by NS007-dev** [reply]
I like how you can clearly see the two letters. You are right. It should only be based in 1 to 2 Color’s. Thank you

**Reply by suitupstan** [reply]
Let's not go behind the mascot colours :) The mascot was made to give EddieHub a friendly, inclusive and abundant look, I think the mascot fulfils this.  A shade of Blue would be psychologically better

**Reply by TakshakRamteke** [reply]
> Let's not go behind the mascot colours :) The mascot was made to give EddieHub a friendly, inclusive and abundant look, I think the mascot fulfils this. A shade of Blue would be psychologically better

@krishguptadev Exactly !

**Reply by suitupstan** [reply]
Ohk, so we need a shade of blue, blending a little bit with purple. IE 30% purple + 70% blue. The blue should be pure blue. The colour should have an optimal light value that would suit both dark and light backgrounds, the colour should have enough saturation to stand out!

**Reply by TakshakRamteke** [reply]
> Ohk, so we need a shade of blue, blending a little bit with purple. IE 30% purple + 70% blue. The blue should be pure blue. The colour should have an optimal light value that would suit both dark and light backgrounds, the colour should have enough saturation to stand out!



you mean something like this

![gradient-logo's](https://user-images.githubusercontent.com/44837950/137746791-b4b7bd7b-52c9-46e9-88f6-3d7a144997ee.png)

**Reply by suitupstan** [reply]
Not that bright blue. Reduce the saturation and brightness of the blue

**Reply by TakshakRamteke** [reply]
> Not that bright blue. Reduce the saturation and brightness of the blue

Do you have some sort of example maybe, That would really help

**Reply by suitupstan** [reply]
Like it shouldn't be too catchy, something like the twitter/google blue works well :thinking:

**Reply by NS007-dev** [reply]
@krishguptadev is this the blue you mean:

![image](https://user-images.githubusercontent.com/70807500/138889048-0438a73a-cc7a-4511-9015-857b292a3351.png)

**Reply by suitupstan** [reply]
> @krishguptadev is this the blue you mean:
> 
> ![image](https://user-images.githubusercontent.com/70807500/138889048-0438a73a-cc7a-4511-9015-857b292a3351.png)

Yeah this also looks good :+1:

**Reply by TakshakRamteke** [reply]
something like this ?

![logo1](https://user-images.githubusercontent.com/44837950/139065368-db04de1e-74b6-4c82-8f9b-a8321c6db197.png)

**Reply by NS007-dev** [reply]
This looks nicer @TakshakRamteke

**Reply by TakshakRamteke** [reply]
mm... how's this one 

![trial4](https://user-images.githubusercontent.com/44837950/139289342-0ac02e41-e788-4e4f-abc2-731e833f4cf0.png)

**Reply by NS007-dev** [reply]
This looks awesome too, I can see the letters more. But, I would say that the size is a bit more bigger, making it more complex if it is to be used as a favicon @TakshakRamteke .

**Reply by NS007-dev** [reply]
I will add it to the poll

**Reply by TakshakRamteke** [reply]
@CodingSpecies Thank you also just for an idea This is How it'd look if we use these as favicon

![favicon-trial4](https://user-images.githubusercontent.com/44837950/139290489-3db208d2-56b1-415d-955a-18e4e2bf8c60.png) 32px


![20px faviccon trial4](https://user-images.githubusercontent.com/44837950/139291028-5ee3e9c2-172e-4260-b0ef-c8374fe6b0fa.png) 20px

**Reply by NS007-dev** [reply]
@TakshakRamteke it actually looks good! Let's see what others say!

**Comment by sakshipawar001** [comment]
Hello everyone. Inspired from @TakshakRamteke and @emmalearnscode , I have come up with another logo design using the initials 'L' and 'F'. Please do review it and give your suggestions on it.
There are two designs attached below. No difference except for the length of second horizontal stroke in letter 'F'. 

![IMG_20211027_204535978](https://user-images.githubusercontent.com/44046769/139211938-54e7c0d6-35a4-47c3-a381-f89e02e3a43e.jpg)

.

**Reply by sakshipawar001** [reply]
Also I have thought of applying linear gradient for color instead of solid colors. If you all are ok with this, I'll share the completed logo for review.

**Reply by NS007-dev** [reply]
@sakshipawar001  this looks interesting. I would really like to see the end result! I think the gradient Color’s would really be cool as well.

**Reply by NS007-dev** [reply]
What colors are you planning to use @sakshipawar001 ?

**Reply by sakshipawar001** [reply]
@CodingSpecies Thank you ! I will soon share the complete logo with you. I have planned to use shades of blue and green.

**Reply by TakshakRamteke** [reply]
@sakshipawar001 Looks really great

**Comment by suitupstan** [comment]
I've tried a few logos:

1:
![Plugin icon - 1](https://user-images.githubusercontent.com/91655303/141669454-8bec7989-9c5a-41c2-a745-4d489d5edc08.png)

2:
![Plugin icon - 1 (1)](https://user-images.githubusercontent.com/91655303/141669455-b5b146e8-b6bf-4659-91c6-a408278b8af7.png)

3:
![Plugin icon - 1 (2)](https://user-images.githubusercontent.com/91655303/141669458-de3f0d8a-f207-4ade-ad8c-e8fe4b38b4f9.png)

4:
![Plugin icon - 1 1](https://user-images.githubusercontent.com/91655303/141669506-e340831e-59be-4834-addd-49e8c78467ab.png)

5:
![Plugin icon - 1 (1) 1](https://user-images.githubusercontent.com/91655303/141669476-ec3824f0-0665-41d1-ab3f-f27a87df99dd.png)

**Reply by TakshakRamteke** [reply]
These one are looking awesome as well krish, 
I personally like the 5th one more :slightly_smiling_face:

**Reply by NS007-dev** [reply]
@krishguptadev I like 2, a lot, but I like how simple yet bold 3 is as well.
Perhaps we can work on the color, but lets see what others say!
Thank you.
I will add this to the poll.

**Comment by TakshakRamteke** [comment]
Hey everyone :wave:, I made this new logo for a change  (because honestly to me the LF one's were felling kinda generic),
also i've tried explained the idea behind this one a little so do let me know your thoughts on this one  and also let me know if i should i explain the idea behind the other logos as well

![g370594](https://user-images.githubusercontent.com/44837950/144054776-91cf96ba-4f66-4410-8d0c-ef56ab954719.png)

**The idea behind this one:**

The inspiration for this one was something like a carbon atoms / fractals (which are beautiful as a one but when combined with multiple thing's (different or same) it becomes something even more beautiful)
Similar to that when a social media is combined with others (of the same person ofcourse xD) it increases the online presence of a indivial which makes them even more awesome. 
which is what LinkFree is usefull for as it hepl's link all the social media link of a person together and Thus the LOGO

`you can think of the centeral hexagon as LinkFree/ user which spreads out and link's to the other social media with those tiny extensions at the each corner of the hexagon`

**Reply by GoodnessEzeh** [reply]
Nice logo with a nicer ideology. Thank you so much, Ramteke

**Reply by suitupstan** [reply]
Looks good!

**Reply by NS007-dev** [reply]
This looks really cool @TakshakRamteke I will add it to the poll!
Thank you.

**Reply by NNANNAJUDE** [reply]
Good idea with wonderful representation. 
Thumbs up!

**Comment by NS007-dev** [comment]
@EddieHubCommunity/linkfree I think now is a good time to finalise our logo, there is plenty to scroll through and take a look at. They are all innovative, practical and brilliant! See here for the poll: https://github.com/EddieHubCommunity/LinkFree/discussions/206#discussioncomment-1465924

This discussion have been open for a long time, and I think we need some finalising! 
Thank you!

**Reply by suitupstan** [reply]
I like @emmalearnscode's suggestion

**Comment by NS007-dev** [comment]
I made some logos myself, and I will be making some more:
![2(1)](https://user-images.githubusercontent.com/70807500/155559837-23e73c75-8fe4-4c7d-aef5-ac81120a9b31.png)
<img src="https://user-images.githubusercontent.com/70807500/155559842-45ddd347-92bd-42cc-bdc0-8c09841d0fe2.png" height=200 />
Feel free to comment and review.

Thank you.

**Comment by NS007-dev** [comment]
Is it possible if I collect everyone's ideas and logos and make a new discussion in the new *polls discussion category*? This should clean up and make it clear to everyone the ideas we have, and make it easier to find the perfect logo.

Thank you.

**Reply by loftwah** [reply]
I'm interested in contributing to this.

**Comment by schmelto** [comment]
I gave some thought to this topic and this is what I came up with.

![image](https://user-images.githubusercontent.com/30869493/210344434-4faf8bbe-5c03-48b1-8f45-00fcafaec044.png)

![image](https://user-images.githubusercontent.com/30869493/210344494-7c87893b-3f4c-4999-8233-1a9c63f77297.png)

![image](https://user-images.githubusercontent.com/30869493/210345405-6ef4ff87-b0ad-47e0-850b-49f566fbb0db.png)


I combined a link icon with two wings which I got from https://uxwing.com.
This is only a design idea.

**Links:**
* https://uxwing.com/hyperlink-icon/
* https://uxwing.com/wing-icon/

**Reply by eddiejaoude** [reply]
This looks great Tom 👍  

Do the icon licenses allow the modification and use?

**Reply by schmelto** [reply]
@eddiejaoude License: All icons are free to use any personal and commercial projects without any attribution or credit. [More Info](https://uxwing.com/license/)

Permitted
All icons on UXWing free to download, and use personal, commercial projects.
Attribution and Credit is NOT required, however, any credit will be much appreciated.
There is no limit to the number of times you can use.
You may modify the resources according to your requirements.

although if we want to go for this design I would recommend to give this design to a designer or someone with design background

**Reply by eddiejaoude** [reply]
Great, thank you for checking the license 👍

**Reply by loftwah** [reply]
This is 🔥

**Reply by loftwah** [reply]
> @eddiejaoude License: All icons are free to use any personal and commercial projects without any attribution or credit. [More Info](https://uxwing.com/license/)
> 
> Permitted All icons on UXWing free to download, and use personal, commercial projects. Attribution and Credit is NOT required, however, any credit will be much appreciated. There is no limit to the number of times you can use. You may modify the resources according to your requirements.
> 
> although if we want to go for this design I would recommend to give this design to a designer or someone with design background

Pardon my ignorance, but why wouldn't we make something ourselves that looks `inspired` by this one? The music industry always does this, and it is never a copyright issue.

**Reply by loftwah** [reply]
![image](https://user-images.githubusercontent.com/19922556/210383042-d55c702d-da9d-471a-bf4b-4e175f57094a.png)
I have access to this with Canva Pro right now.

[Magic Studio Terms](https://www.canva.com/policies/magic-studio-terms/)

Is there any reason I couldn't use this for a project?

**Reply by eddiejaoude** [reply]
I think the proposal is inspired by the icons already. I am not familiar with Canvas so not sure what that feature provides

Either way we would need a vector file for the final logo so it can scale well etc

**Reply by loftwah** [reply]
![210345405-6ef4ff87-b0ad-47e0-850b-49f566fbb0db](https://user-images.githubusercontent.com/19922556/210471803-515c577c-7062-4b7e-ba57-8e03dd0d7de8.svg)
![210344494-7c87893b-3f4c-4999-8233-1a9c63f77297](https://user-images.githubusercontent.com/19922556/210471808-66739883-2966-431a-9657-ebff799f8a34.svg)
![210344434-4faf8bbe-5c03-48b1-8f45-00fcafaec044](https://user-images.githubusercontent.com/19922556/210471810-0dd90831-68ad-4d6c-b174-3aaaddfddb34.svg)

FTFY 🙇‍♂️

**Reply by eddiejaoude** [reply]
wow 😱  thank you!

**Reply by EslemOuederni** [reply]
I really love this 🔥 Great job !

**Comment by timDeHof** [comment]
it would be really cool if it shows to side in the social accounts of our github profile when we add our Linkfree profile like when you add your Linkedin public profile.

**Comment by loftwah** [comment]
![LinkFree (1)](https://user-images.githubusercontent.com/19922556/223322692-bb8da4b9-0bda-4b56-a26b-ec4b60bda5bc.png)

I ended up doing this.

**Reply by NS007-dev** [reply]
@loftwah these designs are really good and look really modern and bold. I think to perhaps make it even better, the colour scheme should be matching the Eddie hub colours?
Thank you.

**Reply by loftwah** [reply]
Yep they can be changed to whatever anyone sees fit. I was feeling edgy that day I guess haha. `#5A5F92`

**Comment by loftwah** [comment]
![linkfree eddiehub io](https://user-images.githubusercontent.com/19922556/223391498-86b3e875-cd26-49fa-b514-d15b78a8a61c.png)

That one inspired this :) I think this could be potentially used.

**Comment by loftwah** [comment]
![LinkFree (2)](https://user-images.githubusercontent.com/19922556/223398421-1dbf0c4e-a59a-4506-91ce-ae172dd5d7e0.png)

I'm just posting these as I go I hope you don't mind.

**Comment by eddiejaoude** [comment]
Logo and colours are now done, so I will close discussion. Thank you everyone, you were awesome!
